G/GANAK/build_tarballs.jl

```diff
@@ -0,0 +1,45 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "GANAK"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/meelgroup/ganak.git", "36fa4b35c8e3cb588a3a76a87624d97ae7f18285")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ganak/
+mkdir build; cd build
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} ..
+make
+install -c ganak -Dt $bindir 
+install -c src/libganak.so.1.0 -Dt $libdir
```

Comment:
Adding some verbosity, to keep track of what we're doing in the log
```suggestion
install -vc ganak -Dt $bindir 
install -vc src/libganak.so.1.0 -Dt $libdir
```